From 7be89ff81ff90fa4525e6ad4af5f7f029a795ad6 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 13 Oct 2015 16:43:24 +0800
Subject: [PATCH] ARM: dts: rk3228: add ion node

Add ion node, with two memory reserved region,
cma-heap and system-heap

Change-Id: Ic6c4da56e0c608135653f901f37adfed7bc7cb15
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3228.dtsi | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/arm/boot/dts/rk3228.dtsi b/arch/arm/boot/dts/rk3228.dtsi
index f3cbc324d1b2..ec89f2644073 100644
--- a/arch/arm/boot/dts/rk3228.dtsi
+++ b/arch/arm/boot/dts/rk3228.dtsi
@@ -115,6 +115,23 @@
 		status = "disabled";
 	};
 
+	rockchip_ion: rockchip-ion {
+		compatible = "rockchip,ion";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		ion_cma: cma-heap {
+			 compatible = "rockchip,ion-heap";
+			 status = "disabled";
+			 rockchip,ion_heap = <4>;
+			 reg = <0x10000000 0x08000000>; /* 128 MB */
+		};
+		system_heap: system-heap {
+			compatible = "rockchip,ion-heap";
+			rockchip,ion_heap = <0>;
+		};
+	};
+
 	rockchip_clocks_init: clocks-init{
 		compatible = "rockchip,clocks-init";
 		rockchip,clocks-init-parent =
-- 
2.35.3

